airflow/providers/http/hooks/http.py

```diff
@@ -268,6 +268,10 @@ def run_with_advanced_retry(self, _retry_args: dict[Any, Any], *args: Any, **kwa
         # TODO: remove ignore type when https://github.com/jd/tenacity/issues/428 is resolved
         return self._retry_obj(self.run, *args, **kwargs)  # type: ignore
 
+    def url_from_endpoint(self, endpoint: str | None) -> str:
```

Comment:
Should we also set it as deprecated?

Reply:
I don't have an answer to this. In the one situation someone might use this hook method internally and if we deprecate (and remove in the future) without replacement it could be not fair, in the other hand I can't find this method useful

Reply:
It's not worth thinking about it when we are just calling another method.